src/main/java/com/faforever/client/chat/ChatUserService.java

```diff
@@ -143,16 +143,11 @@ private void setGameImages(ChatChannelUser chatChannelUser, PlayerBean player) {
       case PLAYING -> uiService.getThemeImage(UiService.CHAT_LIST_STATUS_PLAYING);
       default -> null;
     };
-    Image mapImage;
-    // Do not load coop map preview because they do not exist on API vault
-    if (status != PlayerStatus.IDLE && player.getGame() != null && player.getGame().getGameType() != GameType.COOP) {
-      mapImage = mapService.loadPreview(player.getGame().getMapFolderName(), PreviewSize.SMALL);
-    } else {
-      mapImage = null;
-    }
     chatChannelUser.setStatusTooltipText(i18n.get(status.getI18nKey()));
     chatChannelUser.setGameStatusImage(playerStatusImage);
-    chatChannelUser.setMapImage(mapImage);
+    chatChannelUser.setMapImage(status != PlayerStatus.IDLE && player.getGame() != null
+        ? mapService.loadPreview(player.getGame().getMapFolderName(), PreviewSize.SMALL)
+        : null);
```

Comment:
This is a little long to do a ternary inline might be better for readability to still assign a local variable